File: ObjCExample/ObjCExample/ViewController.m
//
//  ViewController.m
//  ObjCExample
//
//  Created by Meler Paine on 2020/11/5.
//

#import "ViewController.h"
#import "SwiftStaticLibraryExample-Swift.h"
@import SwiftFrameworkExample;

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UILabel *titleLabel;
@property (weak, nonatomic) IBOutlet UILabel *frameworkLabel;

@end

@implementation ViewController

- (void)viewDidLoad {
  [super viewDidLoad];
  NSString *title = [SwiftStaticLibraryExample new].title;
  NSString *frameworkTitle = [SwiftFrameworkExample new].title;
  self.titleLabel.text = title;
  self.frameworkLabel.text = frameworkTitle;
}

@end

File: SwiftExample/SwiftExample/ViewController.swift
//
//  ViewController.swift
//  SwiftExample
//
//  Created by Meler Paine on 2020/11/5.
//

import UIKit
import SwiftStaticLibraryExample
import SwiftFrameworkExample

class ViewController: UIViewController {

  @IBOutlet weak var titleLabel: UILabel!
  @IBOutlet weak var frameworkLabel: UILabel!
  
  override func viewDidLoad() {
    super.viewDidLoad()
    let title = SwiftStaticLibraryExample().title
    let frameworkTitle = SwiftFrameworkExample().title
    titleLabel.text = title
    frameworkLabel.text = frameworkTitle
  }

}

